### PR TITLE
 Ignore all file types being converted as potential outputs 

### LINKED ISSUE #36: Stuck Converting
I attempted three times to convert different image files (JPG and PNG) and each time, after hitting the "Merge" button, the window changed to "Converting" and stayed that way for several minutes before I manually closed the application via the Windows Task Manager.

I'm using Windows 10.

I've deleted the software out of frustration so I can't do any more testing to figure out the cause, but I can provide more information on my system if it would help.

### PR DESCRIPTION
**Pull request by ericadamski**
Fixes #36, #43

Logic now filters out any types that overlap in the conversion types and the files to covert

### CODE REVIEW

File: src/helpers/util.js [hunk 1]
```diff
@@ -45,6 +45,21 @@ const uniqueFiles = (files, newArray) =>
     newArray.filter(file =>
       !files.map(file => file.path).includes(file.path)))
 
+const getUniqueExtensions = files => {
+    const normalizeExtension = name => {
```
Comment (mfix22) [inline src/helpers/util.js]: You could probably declare `normalizeExtension` outside of this function (:

Comment (ericadamski) [reply src/helpers/util.js]: Sorry I didn't make it :(

Comment (mfix22) [reply src/helpers/util.js]: Don't worry about it 😄I was just nitpicking. Update not necessary. Thanks a lot for this bug fix! We are going to release today.

Comment (mfix22) [reply src/helpers/util.js]: Go for it! 👍 that would be massively appreciated. Also thanks so much for writing that test! We really should have been better about doing so

Comment (ericadamski) [reply src/helpers/util.js]: No shame! TDD is great but it takes time that I usually do not have to commit 😂

**Comment by briandennis** [comment]
wooo, thanks @ericadamski!

**Pull request merged**